cypress/page/menu-content.page.ts

```diff
@@ -0,0 +1,19 @@
+class MenuContentPage {
+  private tShirtMenu: string;
+  private menuContentPageURL: string;
+
+  constructor() {
+    this.menuContentPageURL = "http://automationpractice.com/";
+    this.tShirtMenu = "#block_top_menu > ul > li:nth-child(3) > a";
+  }
+
+  public visitMenuContentPage(): void {
+    cy.visit(this.menuContentPageURL);
+  }
+
+  public goToTShirtMenu(): void {
+    cy.get(this.tShirtMenu).click();
+  }
+}
+
+export default new MenuContentPage();
```

Comment:
not sure about this approach, I prefer the one suggested by the guide, export the class and then have a page/index.ts so you can import from the page index in the buy-shirt.cy.ts file